kairon/api/app/routers/bot/integrations.py

```
@@ -0,0 +1,84 @@
+    if is_event_data is True:
+        event.enqueue()
+        return {"message": "Sync in progress! Check logs."}
+    else:
+        raise AppException(is_event_data)
+
+
+
+@router.post("/{provider}/{sync_type}/{bot}/{token}/{execution_id}", response_model=Response)
+async def rerun_sync(
+    provider: CatalogProvider = Path(description="Catalog provider name",
+                                 examples=[CatalogProvider.PETPOOJA.value]),
```

Ruff (0.8.2)

Do not perform function call `Path` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)